SOURCES/consul.sysconfig

```diff
@@ -1 +1,2 @@
-CMD_OPTS="agent -server -config-dir /etc/consul.d -data-dir /var/lib/consul"
+CMD_OPTS="agent -config-dir /etc/consul.d -data-dir /var/lib/consul -ui-dir /var/lib/consul/ui -client 0.0.0.0"
```

Comment:
I would disagree with every agent serving the web ui. Seems like something that should be left in config.json